pyramid_oereb/views/webservice.py

```diff
@@ -339,10 +339,11 @@ def __validate_extract_params__(self):
         if extract_format == 'pdf' and user_requested_geometry:
             raise HTTPBadRequest('Geometry is not available for format PDF.')
 
-        # If PDF is to be produced, always include geometry
-        # (even though the URL shall not contain geometry parameter)
+        # If PDF is to be produced, check if geometry should be included
+        # (even though the URL is not allowed to not contain geometry parameter)
         if extract_format == 'pdf':
-            with_geometry = True
+            if Config.get('print', {}).get('with_geometry', True):
+                with_geometry = True
```

Comment:
This saves you some writing and is I think easier to read.
```suggestion
            with_geometry = Config.get('print', {}).get('with_geometry', True) or with_geometry
```
(by the way if in the config `with_geometry = False' the value for `with_geometry` here can still be `True` depending on what was set above. I don't know if this is intended?)

Reply:
Thanks, I have changed it.

Regarding your question: the "geometry" option is somewhat confusing, because it is allowed for XML/JSON calls, but not for PDFs. But for some PDF calls (xml2pdf), geometry data is needed.
I have updated the comment in the code, hopefully it is clearer like this.